src/form-group.coffee

```diff
@@ -7,11 +7,22 @@ class FormGroupController
     unref = @$scope.$watch(@update)
     @$scope.$on "$destroy", unref
 
+  setParentForm: (ctrl) ->
+    @ngFormCtrl = ctrl
+
+  canValidate: ->
+    inputsReady = @inputs.every (i) -> (i.$dirty and not i.$pending)
+
+    if @ngFormCtrl
+      @ngFormCtrl.$submitted or inputsReady
+    else
+      inputsReady
```

Comment:
@atruskie - looks like the `$$parentForm` thing broke my tests, so I just broke this out to clean things up a smidge.